src/status_im/common/standard_authentication/events.cljs

```diff
@@ -33,6 +33,8 @@
                                             args-with-biometric-btn])
              :on-success     #(rf/dispatch [:standard-auth/on-biometric-success on-auth-success])
              :on-fail        (fn [err]
+                               (rf/dispatch [:standard-auth/authorize-with-password
```

Comment:
I initially added a condition here
```
(when (and err
           (string/includes?
            (string/lower-case
             (:orig-error-message (.-data err)))
            "too many attempts"))
```
to only show password input when too many attempt. But I think user should be able to manually input password when bio-metric fails, even if due to some other reason.

Reply:
yeah makes sense 👍